src/rnanorm/annotation.py

```diff
@@ -0,0 +1,70 @@
+"""Annotation parsing methods."""
+
+import gzip
+import re
+from collections import defaultdict
+
+import pandas as pd
+
+
+def universal_open(file_name, *args, **kwargs):
```

Comment:
Very cool! Would you support this in all the places where we load files? Before doing that, check if Pandas loads compressed files natively.